### PR TITLE
Add page describing how to use AWS push button

### PR DESCRIPTION
**Pull request by rwsu**
The page includes links to startup an EC2 instance in many regions.
    
 It also includes information on how to use the labs.

### CODE REVIEW

**Comment by rwsu** [comment]
The table needs a make over.

File: pages/ec2.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+We have created AWS images that automatically install Kubernetes
+and KubeVirt inside an EC2 instance to help you quickly deploy 
+a trial environment. These images are not meant to be used in 
+production.
+
+To use the images, you should already have an AWS account. The images 
+are free to use but AWS will bill you for instance hours, storage,
+ and associated services unless you are in an AWS trial period.
+
+### Step 1: Launch KubeVirt in Amazon EC2
```
Comment (mindreeper2420) [inline pages/ec2.md]: Set as an `<h2>` `##`, in order to follow semantic HTML usage rules (the title elements renders as an `h1` when the page renders.

File: pages/ec2.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| ap-southeast-2 | Sydney   | HVM      | [ami-047df916b259b17cf](https://console.aws.amazon.com/ec2/home?region=us-east-1#launchAmi=ami-047df916b259b17cf) |
+| ap-south-1     | Mumbai   | HVM      | [ami-0bbcf719fec0d74e9](https://console.aws.amazon.com/ec2/home?region=us-east-1#launchAmi=ami-0bbcf719fec0d74e9) |
+|            |               |          |        |
+| sa-east-1  | Sao Paulo   | HVM      | [ami-0447be57cacc2bff4](https://console.aws.amazon.com/ec2/home?region=us-east-1#launchAmi=ami-0447be57cacc2bff4) |
+|            |               |          |        |
+
+It takes about 5 mins after the EC2 instance is started for the 
+instance to be ready for SSH login.
+
+### Step 2: Walkthrough the KubeVirt labs
```
Comment (mindreeper2420) [inline pages/ec2.md]: Same here - you can update the an `<h2>` (`##`)

**Comment by mindreeper2420** [comment]
@rwsu There are no basic markdown controls to create a responsive table. We can either add a helper to the environment to allow for that, or JavaScript can be written to try and dynamically apply the Bootstrap 4 classes to the table when rendered as an HTML page.

**Comment by rwsu** [comment]
@mindreeper2420 We don't need responsive tables. I was thinking just a nice light border around the cells would be nice. How is this page able to render borders in the table? https://help.github.com/articles/organizing-information-with-tables/

**Comment by mindreeper2420** [comment]
@rwsu those are static images that they've uploaded to the site as examples, but I can look into adding a custom wrapper. The trick is keeping this a pure markdown file that renders correctly on the web, and not a combination html/markdown file that won't be readable outside of the site.

**Pull request merged**